Solutions/ESI - Exchange Security Config and Audit Insights/Analytic Rules/ServerOrientedWithUserOrientedAdministration.yaml

```diff
@@ -0,0 +1,88 @@
+id: 7bce901b-9bc8-4948-8dfc-8f68878092d5
+name: Server Oriented Cmdlet And User Oriented Cmdlet used
+description: |
+  'Detect if a server oriented Cmdlet and a user oriented cmdlet that are monitored are launched by a same user in a same server in a 10 minutes timeframe'
+requiredDataConnectors:
+  - connectorId: ESI-ExchangeAdminAuditLogEvents
+    dataTypes:
+      - Event
+severity: High
+queryFrequency: 5h
+queryPeriod: 5h
+triggerOperator: gt
+triggerThreshold: 0
+status: Available
+tactics:
+  - Exfiltration
+  - Persistence
+  - Collection
+relevantTechniques:
+  - T1020
+  - T1098
+  - T1114
+query: |
+  let timeframe = 1d;
+  let spanoftime = 10m;
+  let threshold = 0;
+  ExchangeAdminAuditLogs 
+    | where TimeGenerated > ago(2 * timeframe)
+    | where UserOriented == ''
```

Comment:
Any reason we are not using isempty(UserOriented) here?

Reply:
Hi @shainw, there is no reason on that. We can change to isempty function. Do you like I pushed the modification ?

Reply:
Added in last push.